django-stubs/contrib/admin/options.pyi

```diff
@@ -107,18 +107,18 @@ class BaseModelAdmin(Generic[_ModelT]):
     def formfield_for_manytomany(
         self, db_field: ManyToManyField, request: HttpRequest, **kwargs: Any
     ) -> ModelMultipleChoiceField | None: ...
-    def get_autocomplete_fields(self, request: HttpRequest) -> Sequence[str]: ...
+    def get_autocomplete_fields(self, request: HttpRequest) -> _ListOrTuple[str]: ...
     def get_view_on_site_url(self, obj: _ModelT | None = ...) -> str | None: ...
     def get_empty_value_display(self) -> SafeString: ...
-    def get_exclude(self, request: HttpRequest, obj: _ModelT | None = ...) -> Sequence[str] | None: ...
+    def get_exclude(self, request: HttpRequest, obj: _ModelT | None = ...) -> _ListOrTuple[str] | None: ...
```

Comment:
Cool! Previously this wasn't considered an error because `str` satisfies `Sequence[str]` 🙄 

```python
    def get_exclude(self, *args: Any, **kwargs: Any) -> str:
        return "a"
```

Reply:
Yes this is precisely what I ran in to: I wrote

```python
ordering = "-date_created"
```

And was surprised when CI passed for mypy but not for system checks..